src/storage/src/hummock/sstable/utils.rs

```diff
@@ -146,3 +146,14 @@ impl TryFrom<u8> for CompressionAlgorithm {
         }
     }
 }
+
+macro_rules! checked_into_u32 {
```

Comment:
The macro looks a little necessary. Why not `v.try_into().expect("{i} u32 overflow")` when `u32` is required? 🤔

Reply:
+1

Reply:
Actually, I received another completely opposite advice earlier. 
Will replace the macro with a function be more acceptable? @MrCroxx @Li0k

Reply:
function might enough